algorithm_store.json

```diff
@@ -16,15 +16,17 @@
         }
       ],
       "arguments": [
-        {
-          "name": "column_name",
-          "type": "string",
-          "description": "The column to compute the mean and sample variance for. The column must be numeric."
-        },
         {
           "name": "organizations_to_include",
           "type": "list",
           "description": "The organizations to include in the task."
+        },
+        {
+          "name": "columns",
+          "type": "list",
+          "description": "The columns to compute the mean and sample variance for. The columns must be numeric. If not provided, all numeric columns are included.",
+          "default_value": null,
+          "has_default_value": true
```

Comment:
I am not an expert on the algorithm store (and we are currently also not using it in the blueberry registry) bit `"has_default_value": true` and set `"default_value: null" seems strange to me. 

After checking this **does** seem to be valid; https://github.com/vantage6/vantage6/pull/1532/files#diff-8ffe09195c95ca63c93df9d3476ad3fd73d043c2893c9862cd85233607eadce0R37-R38

(I merely left this comment here for you information)